CWE/events/Italy ahistorical.txt

```diff
@@ -1426,6 +1482,8 @@ country_event = {
 		
 		ruling_party_ideology = conservative
 		
+		political_reform = first_past_the_post
```

Comment:
Trigger check should be done this way: voting_system = first_past_the_post. political_reform = x or social_reform = x should only be used in the effects scope.